src/main/scala/org/constellation/snapshotstreaming/Configuration.scala

```diff
@@ -14,13 +22,24 @@ class Configuration {
   private val opensearch = config.getConfig("snapshotStreaming.opensearch")
   private val s3 = config.getConfig("snapshotStreaming.s3")
 
-  val nextOrdinalPath: String = config.getString("snapshotStreaming.nextOrdinalPath")
-
-  val httpClientTimeout = Duration(httpClient.getString("timeout"))
-  val httpClientIdleTime = Duration(httpClient.getString("idleTimeInPool"))
-
-  val nodeIntervalInSeconds: Int = node.getInt("retryIntervalInSeconds")
-  val nodeUrls: List[Uri] = node.getStringList("urls").asScala.toList.map(Uri.unsafeFromString)
+  val lastSnapshotPath: Path = Path(config.getString("snapshotStreaming.lastSnapshotPath"))
+  val l0Peers: NonEmptyMap[PeerId, L0Peer] = NonEmptyMap.fromMapUnsafe(
+    SortedMap.from(
+      node.getStringList("l0Peers").asScala.toList.map(decode[L0Peer](_).toOption).map(p => p.get.id -> p.get)
```

Comment:
`p.get` may fail

Reply:
And I want it to fail if it's not decoded well. But I will move the `get` to right after the `decode`. Anyway we could improve the whole configuration reading and starting part, but that would require changing the library I guess and I didn't want to mix it in into this PR. I can create a low prio ticket for that.